solrcloudpy/utils.py

```diff
@@ -55,12 +55,20 @@ def make_request(host, path):
                                         params=resparams,
                                         headers=headers, data=body, timeout=self.timeout)
 
+                if len(servers) > 0 and hasattr(r, 'status_code') and r.status_code != 200:
```

Comment:
`getattr(object, attr, replacement)` will be more succint here instead of the 2 `and` condiftions. So `getattr(r, 'status_code', 0)` should be the equivalent.